manticore/wasm/structure.py

```diff
@@ -442,7 +454,31 @@ def load(cls, filename: str):
                     m.data.append(
                         Data(MemIdx(d.index), convert_instructions(d.offset), d.data.tolist())
                     )
-            # TODO - custom sections (https://www.w3.org/TR/wasm-core-1/#custom-section%E2%91%A0)
+            elif sec_id == SEC_UNK:
+                # WASM module renders all types as `GeneratedStructureData` so `isinstance` doesn't work :(
+                if (
+                    hasattr(section, "name_type")
+                    and hasattr(section, "payload_len")
+                    and hasattr(section, "payload")
+                ):
+                    name_type = section_data.name_type
+                    if name_type == 0:  # module name
```

Comment:
nit: Should we make an `enum` for the values of `name_type`?

Reply:
There are hardcoded constants in the `wasm` module, but it looks like they're out of date relative to the spec. I added a comment pointing to the spec itself; hopefully that will suffice.